github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -33,6 +33,8 @@ presubmits:
           value: k8s-1.25-sig-compute
         - name: KUBEVIRT_CGROUPV2
           value: "true"
+        - name: KUBEVIRT_WITH_ETC_IN_MEMORY
```

Comment:
> > Did we measure the required memory?
> 
> When testing the sig-compute lane last night - it got to about 400M - the default size is 512M

Do we want to set a bigger default value for this to be safe? We could hide that default in a new preset, this way we would need to change it only once?

Reply:
I was going to suggest to stick with the default value for now and only increase it if we see a need to as memory in the CI cluster is an expensive resource.

Reply:
SGTM.